code/datums/brain_damage/severe.dm

```diff
@@ -0,0 +1,221 @@
+		sleep_chance += 15
+	if(owner.drowsyness)
+		sleep_chance += owner.drowsyness + 5
+		if(owner.drowsyness >= 66)
+			owner.drowsyness = 0
+	if(prob(sleep_chance))
+		to_chat(owner, "<span class='warning'>You fall asleep.</span>")
+		owner.Sleeping(60)
+	else if(prob(sleep_chance * 2))
+		to_chat(owner, "<span class='warning'>You feel tired...</span>")
+		owner.drowsyness += 10
+
+/datum/brain_trauma/severe/monophobia
+	name = "Monophobia"
+	desc = "Patient feels sick and distressed when not around other people, leading to potentially lethal levels of stress."
+	scan_desc = "severe monophobia"
+	gain_text = ""
+	lose_text = "<span class='notice'>You feel like you could be safe on your own.</span>"
+	var/stress = 0
+
+/datum/brain_trauma/severe/monophobia/on_gain()
+	..()
+	if(check_alone())
+		to_chat(owner, "<span class='warning'>You feel really lonely...</span>")
+	else
+		to_chat(owner, "<span class='notice'>You feel safe, as long as you have people around you.</span>")
+
+/datum/brain_trauma/severe/monophobia/on_life()
+	..()
+	if(check_alone())
+		stress = min(stress + 0.5, 100)
+		if(stress > 10 && (prob(5)))
+			stress_reaction()
+	else
+		stress -= 4
+
+/datum/brain_trauma/severe/monophobia/proc/check_alone()
+	if(owner.disabilities & BLIND)
+		return TRUE
+	for(var/mob/M in oview(owner, 7))
+		if(!isliving(M)) //ghosts ain't people
+			continue
+		if((istype(M, /mob/living/simple_animal)) || M.ckey)
+			return FALSE
+	return TRUE
+
+/datum/brain_trauma/severe/monophobia/proc/stress_reaction()
+	if(owner.stat != CONSCIOUS)
+		return
+
+	var/high_stress = (stress > 60) //things get psychosomatic from here on
+	switch(rand(1,6))
+		if(1)
+			if(!high_stress)
+				to_chat(owner, "<span class='warning'>You feel sick...</span>")
+			else
+				to_chat(owner, "<span class='warning'>You feel really sick at the thought of being alone!</span>")
+			addtimer(CALLBACK(owner, /mob/living/carbon.proc/vomit, high_stress), 50) //blood vomit if high stress
+		if(2)
+			if(!high_stress)
+				to_chat(owner, "<span class='warning'>You can't stop shaking...</span>")
+				owner.dizziness += 20
+				owner.confused += 20
+				owner.Jitter(20)
+			else
+				to_chat(owner, "<span class='warning'>You feel weak and scared! If only you weren't alone...</span>")
+				owner.dizziness += 20
+				owner.confused += 20
+				owner.Jitter(20)
+				owner.adjustHalLoss(50)
+
+		if(3, 4)
+			if(!high_stress)
+				to_chat(owner, "<span class='warning'>You feel really lonely...</span>")
+			else
+				to_chat(owner, "<span class='warning'>You're going mad with loneliness!</span>")
+				owner.hallucination += 20
+
+		if(5)
+			if(!high_stress)
+				to_chat(owner, "<span class='warning'>Your heart skips a beat.</span>")
+				owner.adjustOxyLoss(8)
+			else
+				if(prob(15) && ishuman(owner))
+					var/mob/living/carbon/human/H = owner
+					var/obj/item/organ/heart/heart = H.internal_organs_by_name["heart"]
+					heart.take_damage(heart.min_bruised_damage)
```

Comment:
No. This makes absolutely no sense. You shouldn't get heart trauma from a fear of loneliness.

Reply:
Makes sense to me.

Reply:
It doesn't make any sense to me. Your heart won't destroy itself from a fear of loneliness. *Maybe* you would faint, which would prevent exposure to it and make you get better. It wouldn't make your heart start dying.